bindata/network/kuryr/005-daemon.yaml

```diff
@@ -0,0 +1,170 @@
+# More info about the template: https://docs.openstack.org/kuryr-kubernetes/latest/installation/containerized.html#generating-kuryr-resource-definitions-for-kubernetes
+
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: kuryr-cni
+  namespace: kuryr
+  annotations:
+# TODO(dulek): Not really sure if we need to hardcode that now or not?
+#    image.openshift.io/triggers: |
+#      [
+#        {"from":{"kind":"ImageStreamTag","name":"node:v3.11"},"fieldPath":"spec.template.spec.initContainers[?(@.name==\"install-cni-plugins\")].image"},
```

Comment:
It seems the triggers are gone from openshiftsdn bindata/network

Reply:
That's why I'm confused - they don't really have an established solution for getting the image name. I don't think that env var thing is going to last.